src/main/java/frc/robot/vision/rawdata/RawVisionData.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.vision.rawdata;
+
+import edu.wpi.first.math.geometry.Pose3d;
+
+public class RawVisionData {
```

Comment:
Why do we call it raw vision data?
Like all the data we use is "raw" so we can just call it vision data (personally I think observation is a better name)